notebooks/Fairness.ipynb

```diff
@@ -0,0 +1,1126 @@
+      "text/plain": [
+       "<Figure size 360x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "region = fairness_results_df.loc[fairness_results_df['Data type']=='Region grouped']\n",
+    "region.groupby(['Prediction column', 'Type'])['f1'].sum().plot.barh(\n",
+    "    color=['orange', 'green', 'grey'], xlabel='Model and data split', title = 'f1',\n",
+    "    xlim = (0.7,1), figsize=(5,10)\n",
+    ");"
+   ]
+  },
```

Comment:
for some reason when I run this it responds with:

`'Rectangle' object has no property 'xlabel'`

I set it underneath and it works: 

`region = fairness_results_df.loc[fairness_results_df['Data type']=='Region grouped']
ax = region.groupby(['Model name', 'Type'])[['precision_score', 'recall_score']].sum().plot.barh(
    color=[[(1,0.5,0,0.5), (0.2,0.8,0.5,0.5), (0,0,0,0.5)], [(1,0.5,0,1), (0.2,0.8,0.5,1), (0,0,0,0.8)]],
    title = 'Precision and recall scores for different region groups',
    xlim = (0.5,1), figsize=(7,5)
);

ax.set_xlabel('Model and data split')
ax.figure.savefig('docs/precrec_regions_incinternational.jpeg', bbox_inches='tight');`

Reply:
also same for plot below

Reply:
*plots